XCUITests/SettingsTest.swift

```diff
@@ -22,38 +22,33 @@ class SettingsTest: BaseTestCase {
         navigator.goto(SettingsScreen)
         let appsettingstableviewcontrollerTableviewTable = app.tables["AppSettingsTableViewController.tableView"]
         appsettingstableviewcontrollerTableviewTable.cells["OpenWith.Setting"].swipeUp()
+        waitforExistence(appsettingstableviewcontrollerTableviewTable.staticTexts["Use Compact Tabs"])
         appsettingstableviewcontrollerTableviewTable.staticTexts["Use Compact Tabs"].swipeUp()
+        waitforExistence(appsettingstableviewcontrollerTableviewTable.staticTexts["Passcode"])
         appsettingstableviewcontrollerTableviewTable.staticTexts["Passcode"].swipeUp()
         appsettingstableviewcontrollerTableviewTable.staticTexts["Show Tour"].swipeUp()
-        wait(for: 5)
+        wait(for: 2)
+        waitforExistence(appsettingstableviewcontrollerTableviewTable.staticTexts["Help"])
         let helpMenu = appsettingstableviewcontrollerTableviewTable.cells["Help"]
         XCTAssertTrue(helpMenu.isEnabled)
         helpMenu.tap()
-        
-        waitForValueContains(app.textFields["url"], value: "support.mozilla.org")
+            waitForValueContains(app.textFields["url"], value: "support.mozilla.org")
         waitforExistence(app.webViews.staticTexts["Firefox for iOS"])
         XCTAssertTrue(app.webViews.staticTexts["Firefox for iOS"].exists)
-        
         let numTabs = app.buttons["Show Tabs"].value
         XCTAssertEqual("2", numTabs as? String, "Sume should be open in a different tab")
     }
 }
-
-
 extension XCTestCase {
```

Comment:
Since we're not using this method if we delete line 30, let's remove this method.